canonical-web-and-design/snapcraft.io
# Remove base64

Fixes #2296

Remove unnecessary base64 icon from CSS.

### QA
- ./run or https://snapcraft-io-canonical-web-and-design-pr-2317.run.demo.haus/
- go to releases page
- make sure that existing dropdowns on channels render correctly
